packages/cli/src/commands/governance/approve.ts

```diff
@@ -11,37 +11,54 @@ export default class Approve extends BaseCommand {
     ...BaseCommand.flags,
     proposalID: flags.string({ required: true, description: 'UUID of proposal to approve' }),
     from: Flags.address({ required: true, description: "Approver's address" }),
+    useMultiSig: flags.boolean({
+      description: 'True means the request will be sent through multisig.',
+    }),
   }
 
-  static examples = ['approve --proposalID 99 --from 0x5409ed021d9299bf6814279a6a1411a7e866a631']
+  static examples = [
+    'approve --proposalID 99 --from 0x5409ed021d9299bf6814279a6a1411a7e866a631',
+    'approve --proposalID 99 --from 0x5409ed021d9299bf6814279a6a1411a7e866a631 --useMultiSig',
+  ]
 
   async run() {
     const res = this.parse(Approve)
     const account = res.flags.from
+    const useMultiSig = res.flags.useMultiSig
     const id = res.flags.proposalID
     this.kit.defaultAccount = account
     const governance = await this.kit.contracts.getGovernance()
-    const governanceApproverMultiSig = await this.kit.contracts.getMultiSig(
-      await governance.getApprover()
-    )
+    const multiSigAddress = useMultiSig ? await governance.getApprover() : ''
+    const governanceApproverMultiSig = useMultiSig
+      ? await this.kit.contracts.getMultiSig(multiSigAddress)
+      : undefined
+    const approver = useMultiSig ? multiSigAddress : account
 
     // in case target is queued
     if (await governance.isQueued(id)) {
       await governance.dequeueProposalsIfReady().sendAndWaitForReceipt()
     }
 
     await newCheckBuilder(this)
-      .isApprover(account)
+      .isApprover(approver)
+      .addConditionalCheck(`${account} is multisig signatory`, useMultiSig, async () =>
+        governanceApproverMultiSig !== undefined
```

Comment:
Could be more readable to check `useMultiSig` here

Reply:
the useMultiSig is included as the condition to run.

compiler forces me to added a check for governanceApproverMultiSig is undefined before using it